tests/foreman/ui/test_dashboard.py

```diff
@@ -186,23 +186,24 @@ def test_positive_task_status(session):
 
 
 @pytest.mark.upgrade
+@pytest.mark.no_containers
 @pytest.mark.run_in_one_thread
 @pytest.mark.skip_if_not_set('clients')
 @pytest.mark.tier3
+@pytest.mark.rhel_ver_match('8')
 @pytest.mark.skipif((not settings.robottelo.repos_hosting_url), reason='Missing repos_hosting_url')
 @pytest.mark.parametrize(
     'repos_collection',
     [
         {
-            'distro': 'rhel7',
-            'SatelliteToolsRepository': {},
-            'YumRepository': {'url': settings.repos.yum_6.url},
-        },
+            'distro': 'rhel8',
+            'YumRepository': {'url': settings.repos.yum_3.url},
```

Comment:
Or this.

Reply:
ah, yes... iirc that is due to the change in the rhel version, there was some problem with repos.yum_6 though I don't recall what it was exactly